episodes/designing-a-light-microscopy-experiment.md

```diff
@@ -0,0 +1,402 @@
+microscopes'.
+
+![The image above is Napari's Kidney (3D + 3Ch) sample image. This was acquired 
+with confocal fluorescence microscopy.](fig/confocal-example.png){alt="Screenshot 
+of Napari's Kidney (3D + 3Ch) sample image" width='60%'}
+
+### Super-resolution
+
+As there are so many [different 'super-resolution' methods
+](https://royalsocietypublishing.org/doi/10.1098/rsta.2021.0110) available, we 
+will only give a brief mention here. The key point is that super-resolution 
+methods provide extremely high resolution images that go beyond the classic 
+limit set by the diffraction of light. Many of these systems are based on 
+modified widefield or confocal microscope setups. This increase in resolution 
+usually comes at the cost of increased complexity in experiment/microscope setup.
+
+### Choosing a method
+
+Choosing a light microscopy method is about finding the simplest and fastest 
+approach that can address your research question. For example, does it provide 
+the resolution you need? Can it provide good images of thick vs thin samples? 
+Does it image a large enough area? Does it image fast enough to provide the 
+temporal resolution you need?... The online [bioimaging guide
+](https://www.bioimagingguide.org/welcome.html) provides a [useful flowchart
+](https://www.bioimagingguide.org/02_Sample_acquisition/Picking.html) to give 
+you some ideas - as well as links to many other great microscopy resources.
+
+If your research question can be solved by a simpler and less costly method, 
+then usually this is the way to go! It's all about choosing the right tool for 
+the job - different approaches will be best for different research questions. 
+For example, you could count cell number with a high-end super-resolution 
+microscope, but this is also possible with a standard widefield which will be 
+simpler and faster. In general, only move to more complex techniques when your 
+research question really requires it.
+
+::::::::::::::::::::::::::::::::::::: challenge 
+
+## Which microscope?
+
+Which light microscopes could be used to answer our research question - 'Does the 
+chemical affect the number, size or shape of cell nuclei over time?'
+
+Think about the points above and make a list of potential answers. It may be 
+best to discuss in a group, so you can share different ideas.
+
+:::::::::::::::::::::::: solution 
+
+Again, there's no one solution here - it depends on which elements of nucleus 
+shape and size we're focusing on. If we're happy with 2D measures, then 
+widefield fluorescence microscopy would be a great option to allow large numbers 
+of nuclei to be imaged over time.
+
+If we're more interested in the precise 3D volume and shape, then confocal 
+fluorescence microscopy would be a better fit. We'd have to be careful with how 
+often we wanted to image the cells vs the speed of acquisition though.
+
+Super-resolution options wouldn't be required here. Cell nuclei can be easily 
```

Comment:
Again, may be worth highlighting what sort of resolution we are talking about in terms of the technique and the size of the nuclei